public/external-text/default.yaml

```diff
@@ -400,55 +412,57 @@ tabs:
             annual:
               displayUnits: mm/year (cum)
               range:
-                min: 10
-                max: 4000
+                min: 200
```

Comment:
Can you update the ticks to make sure that they are a superset of the included range (e.g. add 6000 below). It feels a bit odd to have ticks defined for values outside of the defined range (e.g. < 200), but OTOH it makes it easier to revert to a lower range later on.